### PR TITLE
Add Stimulus in Webpack Encore recipe

### PR DESCRIPTION
**Pull request by tgalopin**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | -

Add Stimulus in Webpack Encore recipe for Symfony UX support

### CODE REVIEW

File: symfony/webpack-encore-bundle/1.0/package.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
     "devDependencies": {
+        "@symfony/stimulus-bridge": "^1.0.0",
         "@symfony/webpack-encore": "^0.31.0",
```
Comment (jmsche) [inline symfony/webpack-encore-bundle/1.0/package.json:4]: Requires ^0.32, right?

File: symfony/webpack-encore-bundle/1.0/assets/controllers/hello_controller.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Controller } from 'stimulus';
+
+/*
+ * This is an example Stimulus controller!
+ *
+ * Any element with with a data-controller="hello" attribute will cause
```
Comment (nicolas-grekas) [inline symfony/webpack-encore-bundle/1.0/assets/controllers/hello_controller.js:6]: double "with"

**Review by fabpot — APPROVED** [review]

**Review by fabpot — COMMENTED** [review]
merge now

**Comment by alexander-schranz** [comment]
@tgalopin @weaverryan any possibility to make this into an own recipe? We in sulu have our own implementation how we recommend implementing [js components](https://github.com/sulu/web-js-twig-demo). And we still want to use webpack encore but it make it harder for our users that they need now remove always the stimulus integration.

**Comment by tgalopin** [comment]
Perhaps what you're looking for is a dedicated entrypoint for your JS components? Let's chart about it in Symfony Devs Slack?

**Comment by alexander-schranz** [comment]
Thx for your time 🙏  again, for the public as discussed we probably will use only `webpack encore js library` and not the bundle at current state as stimilus is to integrated at current state when using the bundle and I understand your point why it is this way.

**Pull request merged**